datafusion/core/src/physical_optimizer/enforce_distribution.rs

```diff
@@ -928,7 +928,7 @@ fn add_roundrobin_on_top(
         // If any of the following conditions is true
         // - Preserving ordering is not helpful in terms of satisfying ordering requirements
         // - Usage of order preserving variants is not desirable
-        // (determined by flag `config.optimizer.bounded_order_preserving_variants`)
+        // (determined by flag `config.optimizer.prefer_existing_sort`)
```

Comment:
this config option was renamed to prefer_existing_sort but some of the comments and variable names refer to the old name